src/main/java/jenkins/plugins/logstash/persistence/ElasticSearchDao.java

```diff
@@ -136,16 +141,27 @@ public String getKey()
   {
     return uri.getPath();
   }
-
+  
+  public String getMimeType() {
+    return this.mimeType;
+  }
+  
+  public void setMimeType(String mimeType) {
+    this.mimeType = mimeType;
+  }
+  
   String getAuth()
   {
     return auth;
   }
 
-  protected HttpPost getHttpPost(String data) {
-    HttpPost postRequest;
-    postRequest = new HttpPost(uri);
-    StringEntity input = new StringEntity(data, ContentType.APPLICATION_JSON);
+  HttpPost getHttpPost(String data) {
+    HttpPost postRequest = new HttpPost(uri);
+    String mimeType = this.getMimeType();
+    // char encoding is set to UTF_8 since this request posts a JSON string
+    StringEntity input = new StringEntity(data, StandardCharsets.UTF_8);
+    mimeType = (mimeType != null) ? mimeType : ContentType.APPLICATION_JSON.toString();
```

Comment:
mimeType will never be null. If you save the global configuration with no content for the mimeType you will get an empty string. Only when you update the plugin without ever saving the configuration it will be null.